packages/route-viewer-overlay/src/RouteViewerOverlay.story.js

```diff
@@ -1,34 +1,76 @@
+/* eslint-disable react/jsx-props-no-spreading */
```

Comment:
Neah, the props were previously set explicitly for `<BaseMap>` and `<RouteViewerOverlay>`.

Reply:
Oh good call don't know why I didn't think of this -- cleaned up in f13469bc7ec8b24af48c993101ec3ca3bb69afcb